frontend/src/pages/Admin/Components/Setting/Invite/Invite.jsx

```diff
@@ -1,8 +1,67 @@
-import React from "react";
+import React, { useState } from "react";
 import { Button2 } from "../../../../../components/util/Button";
 import style from "./Invite.module.scss";
+import Joi from "joi-browser";
+export function Invite() {
+  const [formdata, setFormData] = useState({
+    name: "",
+    email: "",
+  });
+  const [formerrors, setFormErrors] = useState({});
+  const schema = {
+    name: Joi.string().required(),
```

Comment:
we don't have any field that takes `name`. So,  please remove this from the schema